### ISSUE TITLE
Review and fix examples

### ISSUE THREAD

**Issue by jakobj** [jakobj/python-gp#9]

**Comment by jakobj** [comment]
it looks like that example does not use an evolutionary method (no selection of individuals by fitness) but just randomly mutates expressions and performs gradient descent on the constants. how about adapting this to an example that essentially extends the other one by additionally performing gradient descent to find values for leaf nodes?

**Comment by jakobj** [comment]
#20 is now merged. are you working on adapting the other script or should i have a look?

**Comment by jakobj** [comment]
i'll work on the other example script and include the relevant information (we can backprop through the graph, yeah!) in the README

**Comment by mschmidt87** [comment]
Great, thanks. I think it would be great to have that example, I was just too busy with the other stuff.

**Comment by jakobj** [comment]
the original example was fixed. regarding the other example: closing this issue in favor of #59
